shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -310,8 +318,37 @@ - (void)pushRoute:(NSString*)route {
 
 #pragma mark - Loading the view
 
+static UIView* GetViewOrPlaceholder(UIView* existing_view) {
```

Comment:
Nit: for consistency with the rest of the file, could this be an objective-c selector?

Reply:
The selector will make it an instance method. This utility is not tied to any instance of the view controller.

Reply:
Can't this just be a + (UIView*) getViewOrPlaceholder:?

Reply:
`getViewOrPlaceholder:` isn't related in any way to the meta-class either. I'd rather keep it this way.

Reply:
<feel free to ignore me / just water cooler chatting> isn't this generally true for most private objective-c class methods? What's the advantage for this micro-optimization?